nemo/collections/nlp/data/machine_translation/preproc_mt_data.py

```diff
@@ -503,14 +514,23 @@ def train_yttm_tokenizers(
         decoder_tokenizer_vocab_size,
         decoder_tokenizer_coverage,
         global_rank,
+        encoder_training_sample_size=-1,
+        decoder_training_sample_size=-1,
     ):
         encoder_tokenizer_model = None
         decoder_tokenizer_model = None
         os.makedirs(out_dir, exist_ok=True)
 
+        supported_tokenizers = ['yttm', 'sentencepiece']
```

Comment:
Maybe call this `supported_shared_tokenizers`?

Reply:
I have changed it to `supported_train_tokenizers` - also maintains consistency with the method name (`train_tokenizers`).